packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -160,13 +166,31 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
   private get numericFacetBreadcrumbs() {
     return this.breadcrumbManagerState.numericFacetBreadcrumbs.map(
       (breadcrumb) =>
-        this.getBreadcrumbValues(this.formatRangeBreadcrumb(breadcrumb))
+        this.getBreadcrumbValues(
+          this.formatRangeBreadcrumb(breadcrumb, (value) => {
+            const {language} = this.bindings.i18n;
+            return `${value.start.toLocaleString(
+              language
+            )} - ${value.end.toLocaleString(language)}`;
```

Comment:
Could there be unexpected interactions between `language` localization and `formatRangeBreadcrumb`?

I'm not clear on why both need to be used. What does each do?

Also, what if someone did not want a dash between the start and end? (e.g. they want to place a html tag)
Would this be possible?